app/theme/context.tsx

```diff
@@ -78,27 +72,17 @@ export const ThemeProvider: FC<PropsWithChildren<ThemeProviderProps>> = ({
    * themeScheme is the value from MMKV. If undefined, we fall back to the system theme
    * systemColorScheme is the value from the device. If undefined, we fall back to "light"
    */
-  const themeContext: ImmutableThemeContextModeT = useMemo(() => {
-    const t = initialContext || themeScheme || (!!systemColorScheme ? systemColorScheme : "light")
-    return t === "dark" ? "dark" : "light"
+  const themeContext: any = useMemo(() => {
+    const t = initialContext || themeScheme || "light"
+    return t === "light"
   }, [initialContext, themeScheme, systemColorScheme])
 
   const navigationTheme: NavTheme = useMemo(() => {
-    switch (themeContext) {
-      case "dark":
-        return NavDarkTheme
-      default:
-        return NavDefaultTheme
-    }
+    return NavDefaultTheme
   }, [themeContext])
```

Comment:
if you'd like for completeness sake, the dependency array should be updated/empty with the new changes (x3 times in this file)

```suggestion
  }, [])
```